### ISSUE TITLE
Request: Add compile flag for blocky-only

### ISSUE THREAD

**Issue by menip** [Zylann/godot_voxel#306]
My project does not make use of smooth terrain features, and I would like to not include those, both for slightly smaller binary size and mostly to not see smooth terrain nodes while developing (Alternatively, these could be disable via editor settings).

**Comment by Zylann** [comment]
There can be such a flag to compile without, but...

> Alternatively, these could be disable via editor settings

This alternative wont make your build smaller, it can only hide the classes.

**Comment by menip** [comment]
> Also I thought Godot already had an option to hide classes now?

Yep, this is what I was referring to when talk about editor settings.

**Comment by Zylann** [comment]
I just gave a try at excluding smooth voxels from the build. As a result, I also excluded `VoxelLodTerrain` since it's not worth using with blocky voxels at the moment, which means I also excluded `VoxelInstancer`. In the future these two might come back even if smooth voxels are off though, as they are not directly requiring smooth voxels, it's more about how functional they are with blocky ones.

So here is the sizes comparison of an editor build.
```
With smooth voxels:
	Size: 			72,955,904
	Size on disk: 	72,957,952

Without smooth voxels:
	Size: 			72,268,800
	Size on disk: 	72,269,824
```
With the flag turned off, you gain about 0.687 megabytes.

**Comment by menip** [comment]
Might not be worth the energy to maintain this then. 
Could instead potentially provide editor settings feature templates for smooth+blocky, smooth only, blocky only.

Feel free to close issue.

**Comment by Zylann** [comment]
I don't know how to provide a template for editor settings though 🤔
